creation/web_base/condor_startup.sh

```diff
@@ -74,6 +74,11 @@ error_gen=$(gconfig_get ERROR_GEN_PATH "$config_file")
 # Read the knobs coming from the frontend configuration for blackhole detection (GLIDEIN_BLACKHOLE_NUMJOBS and GLIDEIN_BLACKHOLE_RATE)
 glidein_blackhole_numjobs=$(gconfig_get GLIDEIN_BLACKHOLE_NUMJOBS "$config_file")
 glidein_blackhole_rate=$(gconfig_get GLIDEIN_BLACKHOLE_RATE "$config_file")
+if [[ -n "$GLIDEIN_BLACKHOLE_RATE" && "$GLIDEIN_BLACKHOLE_RATE" =~ ^0(\.0*)?$ ]]; then
```

Comment:
Two questions here:
1. Why are we not checking both `glidein_blackhole_numjobs` and `glidein_blackhole_rate` variables here?

2. What does the regex pattern `^0(\.0*)?$` match? Is it any real number?

Reply:
1. Only considering `glidein_blackhole_rate` since we have defined the following, as per our documentation:
```
Rate at which a Glidein is flagged as a black hole and retired. ... When undefined or set to 0.0, the black hole mechanism is disabled. See the blackhole detection section below ...
```

2. The regex pattern only matches `0`, `0.` `0.0`, `0.00...` (all of which means zero)